component-samples/demo/manufacturer/log4j2.xml

```diff
@@ -0,0 +1,12 @@
+<Configuration status="info">
+    <Appenders>
+        <Console name="CONSOLE" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Root level="info">
```

Comment:
We can additionally, log to a file in 'append' mode, and mount that file in the compose yaml. This way, we'll have access to the log file, even after container is destroyed.
We can discuss and consider this for another PR later.

Reply:
Raised an Issue. Will be resolved in future.